spec/emerald/compiler_spec.rb

```diff
@@ -17,4 +17,24 @@
       expect(compiled_lisp).to eq("")
     end
   end
+
+  context "when the contents of the file define a variable" do
+    it "parses the lisp and returns an assigned global variable" do
+      lisp_file_contents = "(def y (- 1 6))"
+      compiled_lisp = Emerald::Compiler.new(lisp_file_contents).compile
+      expect(compiled_lisp).to eq("y = 1.0 - 6.0")
+    end
+
+    it "parses the lisp and returns an assigned local variable" do
+      lisp_file_contents = "(let [x 1 y (+ 3 4)] x)"
```

Comment:
Hmm I can't actually remember why I wanted to use `[]` instead (or at least not well enough to adequately justify it). I was using `()` initially too, and it was most likely a late-night decision to use `[]`instead, because Clojure and maybe it seemed more readable... I'll change it back though because it messes with the list data structure